core/storage/feedback/gae_models.py

```diff
@@ -41,7 +41,6 @@
 
 class FeedbackThreadModel(base_models.BaseModel):
     """Threads for each exploration.
-
```

Comment:
No need to delete these lines -- the usual docstring format is a single-line docstring, followed by a newline, followed by additional explanations. This is because the first line of each docstring is sometimes the target for an automatic doc-generation system.

We sometimes break this rule because the first line can't fit in under 80 characters, but where possible I suggest we stick to it.